src/test/java/hudson/plugins/plot/CSVSeriesTest.java

```diff
@@ -22,47 +22,89 @@
+    private static final String[] LAST_COLUMN_NAME = {"error %", "thing", "error %"};  //the label on the last column
+    private static final String[] LAST_POINT = {"0.37", "42", "0.37"};  //the value of the last data point
 
     public void testCSVSeriesWithNullExclusionValuesSetsDisplayTableFlag() {
-        CSVSeries series = new CSVSeries(FILES[0], null, null, null, true);
-        assertTrue(series.getDisplayTableFlag());
+        CSVSeries series;
+        for (int testfilenum = 0; testfilenum < FILES.length; testfilenum++) {
+            series = new CSVSeries(FILES[testfilenum], null, null, null, true);
+            assertTrue(series.getDisplayTableFlag());
+        }
     }
 
     public void testCSVSeriesWithNoExclusions() {
-        // first create a FilePath to load the test Properties file.
-        File workspaceDirFile = new File("target/test-classes/");
-        FilePath workspaceRootDir = new FilePath(workspaceDirFile);
+        for (int testfilenum = 0; testfilenum < FILES.length; testfilenum++) {
+            // first create a FilePath to load the test Properties file.
+            File workspaceDirFile = new File("target/test-classes/");
+            FilePath workspaceRootDir = new FilePath(workspaceDirFile);
 
-        LOGGER.info("workspace File path: " + workspaceDirFile.getAbsolutePath());
-        LOGGER.info("workspace Dir path: " + workspaceRootDir.getName());
+            LOGGER.info("workspace File path: " + workspaceDirFile.getAbsolutePath());
+            LOGGER.info("workspace Dir path: " + workspaceRootDir.getName());
 
-        // Check the number of columns
-        int columns = -1;
+            // Check the number of columns
+            int columns = -1;
 
-        try {
-            columns = getNumColumns(workspaceRootDir, FILES[0]);
-        } catch (IOException e) {
-            assertFalse(true);
-        } catch (InterruptedException e) {
-            assertFalse(true);
+            try {
+                columns = getNumColumns(workspaceRootDir, FILES[testfilenum]);
+            } catch (IOException e) {
+                assertFalse(true);
+            } catch (InterruptedException e) {
+                assertFalse(true);
+            }
+
+            assertEquals(COLUMNS[testfilenum], columns);
+
+            // Create a new CSV series.
+            CSVSeries series = new CSVSeries(FILES[testfilenum], "http://localhost:8080/%name%/%index%/", "OFF", "", false);
+
+            LOGGER.info("Created series " + series.toString());
+            // test the basic subclass properties.
+            testSeries(series, FILES[testfilenum], "", "csv");
+
+            // load the series.
+            List<PlotPoint> points = series.loadSeries(workspaceRootDir, 0, System.out);
+            LOGGER.info("Got " + points.size() + " plot points");
+            testPlotPoints(points, TOTAL_POINTS[testfilenum]);
+
+            int pointnum = 0;
```

Comment:
Please follow camelCase in variables naming.
-> `pointNumber`
-> `columnNumber`